pool/pool.py

```diff
@@ -596,7 +616,7 @@ async def add_farmer(self, request: PostFarmerRequest, metadata: RequestMetadata
             if len(hexstr_to_bytes(request.payload.payout_instructions)) != 32:
```

Comment:
This should also validate the payout instructions in the same way.

Reply:
Fixed